target/scripts/startup/setup.d/getmail.sh

```diff
@@ -4,38 +4,59 @@ function _setup_getmail() {
   if [[ ${ENABLE_GETMAIL} -eq 1 ]]; then
     _log 'trace' 'Preparing Getmail configuration'
 
-    local GETMAILRC ID CONFIGS
+    # Verify correct value for GETMAIL_POLL. Valid are any numbers greater than 0.
+    if ! [[ ${GETMAIL_POLL} =~ ^[0-9]+$ && ${GETMAIL_POLL} -gt 0 ]]; then
+      _log 'warn' "Invalid value for GETMAIL_POLL: ${GETMAIL_POLL}"
+      _log 'warn' "Getmail will be disabled"
+      # return 1
+    fi
 
-    GETMAILRC='/etc/getmailrc.d'
-    CONFIGS=0
+    local GETMAIL_RC ID GETMAIL_DIR
 
-    mkdir -p "${GETMAILRC}"
+    local GETMAIL_CONFIG_DIR='/tmp/docker-mailserver/getmail'
+    local GETMAIL_RC_DIR='/etc/getmailrc.d'
+    local GETMAIL_RC_GENERAL_CF="${GETMAIL_CONFIG_DIR}/getmailrc_general.cf"
+    local GETMAIL_RC_GENERAL='/etc/getmailrc_general'
 
-    # Generate getmailrc configs, starting with the `/etc/getmailrc_general` base config,
-    # Add a unique `message_log` config, then append users own config to the end.
-    for FILE in /tmp/docker-mailserver/getmail-*.cf; do
-      if [[ -f ${FILE} ]]; then
-        CONFIGS=1
-        ID=$(cut -d '-' -f 3 <<< "${FILE}" | cut -d '.' -f 1)
-        local GETMAIL_CONFIG="${GETMAILRC}/getmailrc-${ID}"
+    # Create the directory /etc/getmailrc.d to place the user config in later.
+    mkdir -p "${GETMAIL_RC_DIR}"
```

Comment:
`debug-getmail.sh` depends on that directory to exist. So it should be created, even when errors occur. This enables the user to debug even in case of errors.

Reply:
> This enables the user to debug even in case of errors.

I thought that the usage to loop over `/etc/getmailrc.d/*` with nullglob set avoids any issues? Doesn't cover the scenario when the path is invalid? Fair enough 👍 

The bailing conditions are:
- `GETMAIL_POLL` is invalid. Although I suggested removing that from `getmail.sh` due to `getmail-service.sh` repeating the logic.
- Config volume `getmail/` dir has no account configs provided (`--rcfile` needs that to exist)

If `--rcfile` cannot be satisfied, then it would not matter if `--getmaildir` cannot be satisfied? (_the other `mkdir` for `/var/lib/getmail`_)

Reply:
> Although I suggested removing that from getmail.sh due to getmail-service.sh repeating the logic.

Done.

> If --rcfile cannot be satisfied, then it would not matter if --getmaildir cannot be satisfied?

Correct.